client/src/context/ProfileContext.js

```diff
@@ -0,0 +1,168 @@
+import React, { useReducer } from 'react';
+import axios from 'axios';
+
+
+//Created Context
+export const ProfileContext = React.createContext();
+
+
+//Stored action types names
+
+const types = {
+	Add_Profile:"Add_Profile",
+	Get_Profile: "Get_Profile",
+	Get_Profiles: "Get_Profiles",
+	Delete_Profile: "Delete_Profile",
+	Edit_Profile: "Edit_Profile",
+	Profile_Error: "Profile_Error",
+	Clear_Profile :"Clear_Profile",
+	Open_Edit :"Open_Edit",
+};
+
+
+//Stored actions in a reducer
+
+const profileReducer = (state, action) => {
+	switch (action.type) {
+	case types.Profile_Error:
+		return { ...state, errorMessage: action.payload, loading: false };
+	case types.Add_Profile:
```

Comment:
I can't see `errorMessage` and `loading` state anywhere - are these being used yet?